packages/account/src/Types/common-prop.types.ts

```diff
@@ -0,0 +1,63 @@
+/** Add types that are shared between components */
+
+type TradingAccountDetail = {
+    linked_to: {
+        account_id: string;
+        balance: string;
+        currency: string;
+        payment_method: string;
+    };
+};
+
+export type TAuthAccountInfo = {
+    account_type: 'trading' | 'wallet';
+    balance: number;
+    country: string | undefined;
+    created_at: number;
+    currency: string;
+    email: string;
+    is_disabled: 0 | 1;
```

Comment:
Just for my understanding, why we don't use `boolean` type here?

Reply:
It should have. But the data that is being sent is 0 or 1, not true or false